includes/db/Database.php

```diff
@@ -3720,6 +3725,10 @@ protected function logSql( $sql, $ret, $fname, $elapsedTime, $isMaster ) {
 			'exception'   => new Exception(), // log the backtrace
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)